### DISCUSSION TITLE
Warning message on [MultiAppGeneralFieldShapeEvaluationTransfer]

### DISCUSSION THREAD

**Discussion by Bearunner** [idaholab/moose#26846] [Q&A General] [answered]
Dear all,

I get the following warning message when I use the [MultiAppGeneralFieldShapeEvaluationTransfer].

*** Warning ***
The following warning occurred in the object "to_sub", of type "MultiAppGeneralFieldShapeEvaluationTransfer".

On rank 0, multiple valid values from equidistant points were received for target variable 'a_res' for 1 target points.
Are some points in target mesh equidistant from the sources (nodes/centroids/apps/positions, depending on transfer) in origin mesh(es)?
Conflicts detected at :
In target child app 0 mesh, point: (-475.000000, -10.000000, -10.000000), equi-distance: 0.000000

How can I solve this warning problem?  

Thanks a lot.

B

**Comment by GiudGiud** [comment]
Hello

How many source applications are there?
It seems they overlap on that target point

Guillaume

**Reply by Bearunner** [reply]
Hi Guillaume,

Only one main app (3D cubic) and one sub app (1D line). The  3D cubic is centrally crossed by the 1D line. The transfer happens from the variable 'a' of the 3D main app to the aux variable 'a_res' of the 1D sub app as shown in the upper figure.

**Reply by GiudGiud** [reply, marked as answer]
How many processes? on the main app and on the sub app.

I think maybe the target location `(-475.000000, -10.000000, -10.000000)` lies at the boundary between two elements in the main app?

You can use `search_value_conflicts=false` to silence it if the value received is consistently correcty.

**Reply by Bearunner** [reply]
Thanks for your tips. It works well so I can turn off this search_value_conflicts.
